components/area_office_map.js

```diff
@@ -162,12 +177,33 @@ const mapStateToProps = reduxState => {
 };
 
 AreaOfficeMap.propTypes = {
+  /**
+   * List of area offices
+   */
   areaOffices: PropTypes.array.isRequired,
+  /**
+   * Sets a selected area office in redux
+   */
   setSelectedAreaOffice: PropTypes.func.isRequired,
+  /**
+   * Sets properties of the map view (Lat, Lng, Zoom)
+   */
   setMapView: PropTypes.func.isRequired,
+  /**
+   * The selected area office
+   */
   selectedAreaOffice: PropTypes.object.isRequired,
+  /**
+   * Properties of the map view (Lat, Lng, Zoom)
+   */
   mapView: PropTypes.object.isRequired,
+  /**
+   * Object that contains CSS classes
+   */
   classes: PropTypes.object.isRequired,
+  /**
+   * Universal translation function
```

Comment:
This is definitely cool, but it will be very time consuming to add these comments to every prop in every component. Will the documentation still generate for files that don't have these comments?

Reply:
yes it just looks like this 

<img width="1032" alt="screen shot 2018-08-23 at 11 54 34 am" src="https://user-images.githubusercontent.com/867334/44536839-5b971500-a6cb-11e8-8a6e-b1d65174ba4b.png">

So it is pretty nice out of the box